src/components/figures/figure_chart/figure_chart.ts

```diff
@@ -50,7 +50,7 @@ export class ChartFigure extends Component<Props, SpreadsheetChildEnv> {
       action: async () => {
         this.env.model.dispatch("SELECT_FIGURE", { id: this.props.figure.id });
         this.env.model.dispatch("COPY");
-        await this.env.clipboard.clear();
```

Comment:
TBH I don't know why we need `clear()`here, so I deleted it.

Reply:
It's supposed to do the same thig you do (` getters.getClipboardContent()` is empty when copying charts), but it seems that the implementation of `clipboard.clear()` doesn't work. 

Well it make more sense to write the content of our internal clipboard rather than clear it so it was the same value as our empty clipboard (\t) You can probaly entierly remove `.clear` from the clipboard in the env, I don't think we were using it elsewhere.